src/views/AccountPage.vue

```diff
@@ -102,8 +91,7 @@ export default defineComponent({
   data (): AccountPageData {
     return {
       operations: [],
-      name: '',
-      editingTitle: false
+      amountInOperation: 0
```

Comment:
? not sure what purpose it serves. Which operation are we talking about ?

Reply:
So... actually, it doesn't use... I will remove it